docs/ml-classification-regression.md

```diff
@@ -97,10 +97,6 @@ only available on the driver.
 [`LogisticRegressionTrainingSummary`](api/scala/index.html#org.apache.spark.ml.classification.LogisticRegressionTrainingSummary)
 provides a summary for a
 [`LogisticRegressionModel`](api/scala/index.html#org.apache.spark.ml.classification.LogisticRegressionModel).
-Currently, only binary classification is supported and the
```

Comment:
Should we add a note reflecting the difference between the summary and binary summary? Perhaps indicating the usage of `binarySummary` or `asBinary` method?

I know it's done in the example but perhaps a short line about that.

Reply:
Done.